pints/_evaluation.py

```diff
@@ -413,6 +413,41 @@ def clear(q):
         return errors
 
 
+class MultiSequentialEvaluator(Evaluator):
+    """
+    Evaluates a list of functions (or callable objects) for a list of input
+    values of the same length, and returns a list containing the calculated
+    function evaluations.
```

Comment:
Although this is clear, it can be useful to give an example below and explain when we want to use this instead of the other two evaluators.

Reply:
I know this might not be used by most of the users, but even when I first looked at it, I had to look at the code to remind myself why we wanted this.